faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -451,13 +487,20 @@ private static URI appendUri(URI oldUri, String queryKey, List<String> queryValu
             oldUri.getQuery() == null ? query : oldUri.getQuery() + "&" + query, oldUri.getFragment());
   }
 
-  private CompletableFuture<HttpResponse<String>> performRequest(String httpMethod, String path, Optional<JsonNode> body,
-                                                                 Map<String, List<String>> params, final Optional<Duration> requestQueryTimeout) {
+  private CompletableFuture<HttpResponse<String>> performRequest(String httpMethod,
+                                                                 String path,
+                                                                 Optional<JsonNode> body,
+                                                                 Map<String, List<String>> params,
+                                                                 final Optional<Duration> requestQueryTimeout,
+                                                                 final Optional<String> requestTraceId,
+                                                                 final Optional<Map<String, String>> requestTags) {
```

Comment:
the rest of this package doesn't really use final much. we should probably stick with convention.

Reply:
Updated